lib/node_modules/@stdlib/string/base/replace-after/lib/main.js

```diff
@@ -0,0 +1,73 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MAIN //
+
+/**
+* Replaces the substring after the first occurrence of a specified search string.
+*
+* @param {string} str - input string
+* @param {string} search - search string
+* @param {string} replacement - replacement string
+* @param {integer} [fromIndex=0] - index at which to start the search
```

Comment:
@AuenKr As mentioned in https://github.com/stdlib-js/stdlib/pull/1363#issuecomment-1962324838, would you mind updating to make `fromIndex` a mandatory argument? Hence, `fromIndex` should not have a default value, but should always be provided by end-users.

The rationale is that this is a "base" package and we try to ensure that base packages are not variadic in order to avoid potential compiler de-optimizations.